packages/desktop-client/src/components/accounts/TransactionsTable.js

```diff
@@ -271,7 +271,11 @@ function getPayeePretty(transaction, payee, transferAcct) {
   let { payee: payeeId } = transaction;
 
   if (transferAcct) {
-    const Icon = transaction.amount > 0 ? LeftArrow2 : RightArrow2;
+    const Icon =
+      (transaction._inverse ? -1 : 1) * transaction.amount > 0
```

Comment:
I feel like this would be clearer if the two branches of the ternary did the comparison directly instead of negating the amount, but I also see the value in being similar to the other places the `_inverse` prop is checked.

Reply:
Is this what you meant? I think it reads worse

```
    const Icon = transaction._inverse
      ? transaction.amount > 0
        ? RightArrow2
        : LeftArrow2
      : transaction.amount > 0
      ? LeftArrow2
      : RightArrow2;
```

Reply:
Yeah, I don’t know what I was thinking.